source/base/Atomic.ooc

```diff
@@ -0,0 +1,56 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use system
```

Comment:
Just stopping by because I was curious about your implementation and because I like commenting on stuff.
You don't need to explicitly import `system`.
It's also OK to write the methods below as one-liners.
See ya.